config/test.yaml

```
@@ -8,4 +8,7 @@ aws: {
     bucket: 'dluhc-data-platform-request-files-local',
     endpoint: 'http://localhost:4566',
     s3ForcePathStyle: true,
+}
+googleAnalytics: {
+  measurementId: 'G-TEST-CODE'
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)